.gitignore

```diff
@@ -34,3 +36,4 @@ TAGS
 # config files
 *.config
 !build.config
+!build.iotivity.config
```

Comment:
1. Remove this end of line character.
2. We didn't talk about using another config file. It's duplicated. Should this file exist separately? Don't you add it on the build.config file?